### ISSUE TITLE
Synchronization error when value from runner's delay field is empty

### ISSUE THREAD

**Issue by dyymmi** [a85/POSTMan-Chrome-Extension#3701]
## App Details:

```
Postman for Windows
Version 5.3.2
win32 10.0.14393 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: **Recently but can't guarantee it wasn't there before**
 2. Expected behaviour: **App should not allow to leave *delay* field empty or handle this under the hood**
 3. Console logs:
```
Entity: collectionrun
Action: create
Request Size: 7.73 KB
Error: {
    "name": "WLError",
    "message": "1 attribute is invalid",
    "details": "Invalid attributes sent to CollectionRun:\n • delay\n   • `undefined` should be a integer (instead of \"NaN\", which is a number)\n"
}
```

## Steps to reproduce the problem:

1. Open Runner
2. Delete value from delay field
3. Run collection
4. Wait for synchronization 

Popup with synchronization error appears with a log added above.

**Comment by prashantagarwal** [comment]
@dyymmi Thanks for reporting this. Were able to reproduce. We will release the fix in upcoming version.

**Comment by prashantagarwal** [comment]
@dyymmi We have fixed this internally. It will be available in next release. From now onwards if you don't provide any delay it will take it as 0ms

**Comment by prashantagarwal** [comment]
@dyymmi This is fixed in Postman 5.4.0+ versions. Feel free to reopen if you are still seeing this issue.
